src/fidesctl/cli/utils.py

```diff
@@ -144,43 +145,45 @@ def send_init_analytics(opt_out: bool, config_path: str, executed_at: datetime)
         pass  # cli analytics should fail silently
 
 
-def with_analytics(ctx: click.Context, command_handler: Callable, **kwargs: Dict) -> Any:  # type: ignore
+def with_analytics(func: Callable) -> Callable:
     """
-    Send an `AnalyticsEvent` with details about the executed command,
+    Click command decorator which can be added to enable publishing anaytics.
+    Sends an `AnalyticsEvent` with details about the executed command,
     as long as the CLI has not been configured to opt out of analytics.
 
-    :param ctx: The command's execution `click.Context` object
-    :param command_handler: The handler function defining the evaluation logic for the analyzed command
-    :param **kwargs: Any arguments that must be passed to the `command_handler` function
+    :param func: function to be wrapped by decorator
     """
 
-    command = " ".join(filter(None, [ctx.info_name, ctx.invoked_subcommand]))
-    error = None
-    executed_at = datetime.now(timezone.utc)
-    status_code = 0
+    def new_func(ctx: click.Context, *args, **kwargs) -> Any:  # type: ignore
```

Comment:
Opinion here but thinking there may be something more descriptive we can use besides `new_func`?

Reply:
sure, renamed it to `wrapper_func`. Lmk if that's not okay!

Reply:
sounds good, thanks!